lib/ruby_indexer/lib/ruby_indexer/configuration.rb

```diff
@@ -24,7 +24,15 @@ def initialize
       @workspace_path = T.let(Dir.pwd, String)
       @excluded_gems = T.let(initial_excluded_gems, T::Array[String])
       @included_gems = T.let([], T::Array[String])
-      @excluded_patterns = T.let([File.join("**", "*_test.rb"), File.join("tmp", "**", "*")], T::Array[String])
+
+      @excluded_patterns = T.let(
+        [
+          File.join("**", "*_test.rb"),
```

Comment:
Let's take advantage of the change and exclude the `test` and `spec` directories completely.
```suggestion
          File.join("test", "**", "*"),
          File.join("spec", "**", "*"),
```

Reply:
Should we exclude `sorbet/**/*` while we're at it? At my job, `sorbet` directory is 1331 files. `tmp` is 146k files, which is why this change helps so much. Our `spec` directory is 558 files.